zulipterminal/ui_tools/views.py

```diff
@@ -968,14 +968,20 @@ def __init__(self, controller: Any, color: str,
 
 
 class MsgInfoView(PopUpView):
-    def __init__(self, controller: Any, msg: Message, title: str) -> None:
+    def __init__(self, controller: Any, msg: Message, title: str,
+                 message_links: List[Tuple[str, str]]) -> None:
         self.msg = msg
 
         msg_info = [
             ('', [('Date & Time', time.ctime(msg['timestamp'])[:-5]),
                   ('Sender', msg['sender_full_name']),
                   ('Sender\'s Email ID', msg['sender_email'])]),
         ]
+        if message_links:
+            msg_info.append(('Message Links (see footlinks)', [
+                (str(index), text)
```

Comment:
I attempted adding `text` and `link` but they disrupt `MsgInfoView` and trigger the wide column issue that we decided to tackle later in https://github.com/zulip/zulip-terminal/pull/659#discussion_r434257471. That said, even if we fix the issue, we'd end up with very wide `MsgInfoView` mostly.

I have added `index` and `text`, for now, but they don't seem to add much value.

Would it be desirable to introduce a message links popup inside `MsgInfoView` (like the edit history popup)?